lib/ramble/ramble/language/modifier_language.py

```diff
@@ -35,7 +35,7 @@ def _execute_mode(mod):
     return _execute_mode
 
 
-@modifier_directive('default_modes')
+@modifier_directive(dicts=())
```

Comment:
What do the empty parentheses mean here? Is this meant to be an empty dictionary?

Reply:
No, the parentheses make a tuple. But, it's empty, so it is effectively an empty object with low overhead.